tests/test_msg_builder.rs

```diff
@@ -0,0 +1,53 @@
+extern crate serenity;
+
+use serenity::utils::MessageBuilder;
+use serenity::model::Emoji;
+use serenity::model::EmojiId;
+use serenity::model::UserId;
+
+#[test]
+fn code_blocks() {
+	let content = MessageBuilder::new()
+		.push_codeblock("test", Some("rb"))
+		.build();
+	assert_eq!(content, "```rb\ntest\n```".to_string());
+}
+
+
+#[test]
+fn safe_content() {
+	let content = MessageBuilder::new()
+		.push_safe("@everyone discord.gg/discord-api")
+		.build();
+	assert!(content != "@everyone discord.gg/discord-api".to_string());
```

Comment:
Strings can be directly compared to slices, so the `.to_string()` isn't needed here. This can then be an `assert_eq`